### PR TITLE
Bugs

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/services/com/k_int/kbplus/SubscriptionUpdateService.groovy [hunk 1]
```diff
@@ -51,14 +51,16 @@ class SubscriptionUpdateService extends AbstractLockableService {
...
+                status == RDStore.SUBSCRIPTION_INTENDED &&
+                        ((instanceOf != null && instanceOf.startDate < currentDate && instanceOf.endDate != null && instanceOf.endDate >= currentDate)
+                                || (instanceOf == null && startDate < currentDate && endDate != null && endDate >= currentDate)) && isMultiYear == true
             }.collect{ it.id }
 
             log.info("Intended perennial subscriptions reached start date and are now running (${currentDate}): " + intendedSubsIds2)
 
             if (intendedSubsIds2) {
-                updatedObjs << ['MultiYear_intendedToCurrent' : intendedSubsIds2]
+                updatedObjs << ["MultiYear_intendedToCurrent (${intendedSubsIds2.size()?:0})" : intendedSubsIds2]
```
Comment (croqueGrec09) [inline app/grails-app/services/com/k_int/kbplus/SubscriptionUpdateService.groovy:63]: No need to make null check here; the check is done one line above.

File: app/web-app/setup/PropertyDefinition.csv [hunk 2]
```diff
@@ -221,8 +221,8 @@ Subscription Property,Time of billing,Rechnungszeitpunkt,Time of billing,class j
...
-Survey Property,Multi-year term 2 years,Mehrjahreslaufzeit 2 Jahre,Multi-year term 2 years,class com.k_int.kbplus.RefdataValue,y.n,,,,Bitte geben Sie hier an,ob Sie eine Lizenzierung direkt für zwei Jahre wünschen.,Please indicate here,if you wish a licensing directly for two years.,,
-Survey Property,Multi-year term 3 years,Mehrjahreslaufzeit 3 Jahre,Multi-year term 3 years,class com.k_int.kbplus.RefdataValue,y.n,,,,Bitte geben Sie hier an,ob Sie eine Lizenzierung direkt für drei Jahre wünschen.,Please indicate here,if you wish a licensing directly for three years.,,
+Survey Property,Access choice remote,Zugangswahl Remote,Access choice remote,class com.k_int.kbplus.RefdataValue,access.choice.remote,,,,"Bitte geben Sie hier an, ob Sie 2FA,Zugang für Wissenschaftler oder kein remote Zugang wünschen?","Please indicate here whether you want 2FA,access for scientists or no remote access?",
```
Comment (croqueGrec09) [inline app/web-app/setup/PropertyDefinition.csv:224]: What is 2FA?

Comment (Moetez16) [reply app/web-app/setup/PropertyDefinition.csv:224]: Zwei-Faktor-Authentisierung

Comment (croqueGrec09) [reply app/web-app/setup/PropertyDefinition.csv:224]: Okay

**Pull request merged**